docs/tables/oci_advanced_resource_query_search.md

```diff
@@ -0,0 +1,49 @@
+# Table: oci_advanced_resource_query_search
+
+OCI advanced resource query search lets you query any and all compartments in the specified tenancy to find resources that match the specified criteria.
+
+**You must specify a Query** in a `where` clause (`where query='`).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  identifier,
+  display_name,
+  time_created,
+  lifecycle_state as state
+from
+  oci_advanced_resource_query_search
+where
+  query = 'query all resources';
```

Comment:
@bigdatasourav let's cut this off to less output query.  We can make a specific call e.g 

```sql
select
  identifier,
  display_name,
  time_created,
  lifecycle_state as state
from
  oci_advanced_resource_query_search
where
  query = `query instance resources
    where lifeCycleState = 'RUNNING'`
```